<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Inbound Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>1581</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTrigger">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trigger</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Ready  </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Running</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string> Alarm  </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbReadyLight_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(178, 190, 181);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lbReadyLight_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(178, 190, 181);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbReadyLight_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(178, 190, 181);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLabel" name="lbl_ram">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RAM</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_fps">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbl_cpu">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FPS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_clock">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0)</string>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1231</width>
      <height>911</height>
     </rect>
    </property>
    <property name="title">
     <string>Main View</string>
    </property>
    <widget class="QLabel" name="label_camera">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>1130</width>
       <height>635</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 228, 226);</string>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>700</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Optimize Image</string>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Exposure (us)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>us</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gain</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>699</y>
       <width>901</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Statistics</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>361</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>products/h</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>NG</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcdNumber_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>29</y>
        <width>421</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="1">
        <widget class="QLCDNumber" name="lcdNumber_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Main Conveyor Speeds:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Weight Conveyor Speeds:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>mm/s</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>mm/s</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>80</y>
      <width>341</width>
      <height>911</height>
     </rect>
    </property>
    <property name="title">
     <string>Result Panel</string>
    </property>
    <widget class="QLabel" name="lbResult1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>320</width>
       <height>320</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 228, 226);</string>
     </property>
     <property name="text">
      <string>Result 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>370</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Width (mm):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_6"/>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcdNumber_7"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Length (mm):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
